### PR TITLE
Add Spider-Ham and SP//DR Suit

### PR DESCRIPTION
**Pull request by funkymonkeymonk**
Adding new heroes and cards from new hero packs. Also adding instructions on how to do it from @SwedeRacer .

### CODE REVIEW

**Comment by funkymonkeymonk** [comment]
There is some issues on the MarvelCDB side that are preventing decks from building.

**Comment by funkymonkeymonk** [comment]
Following https://github.com/zzorba/marvelsdb-json-data/pull/272

**Comment by funkymonkeymonk** [comment]
So this looks done from our side. The only thing we're waiting for is to have the MarvelCDB JSON data corrected. Otherwise we are good to go.

File: mod/src/MarvelChampionsLCG/Hero_Bag/SP_dr_Hero_Bag.283667.lua [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  heroDetails.heroDeckId = 356173
+  heroDetails.nemesisGuid = "49bccd"
+  heroDetails.obligationGuid = "ba1312"
+  heroDetails.hitPoints = 14
+  heroDetails.counterUrl = "http://cloud-3.steamusercontent.com/ugc/1897721393463027391/971F5CA033F0BD7AFF157E3716D3BD6B745A946C/"
+  heroDetails.playmatUrl = "http://cloud-3.steamusercontent.com/ugc/1861691130254392381/73723523D39B54A9710C85B5D8CF6DE445E4E317/"
+
+  local extras = {}
+
+  extras[""] =
```
Comment (SwedeRacer) [inline mod/src/MarvelChampionsLCG/Hero_Bag/SP_dr_Hero_Bag.283667.lua:15]: Even if it's not strictly necessary, it's probably a good idea to add a descriptive key to every extra, just to make it easier to identify what it represents in the code.

**Pull request merged**